module/VuFind/sql/migrations/mysql/11.0/008-add-access-token-expires-boolean.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE `access_token` ADD COLUMN `expires` tinyint(1) NOT NULL DEFAULT '1';
```

Comment:
These migrations need to be renumbered now. If you think we're close to being ready to merge, feel free to do it now; if you'd rather wait until more things are finalized, that's fine as well.